.circleci/config.yml

```diff
@@ -47,12 +44,9 @@ jobs:
       - run:
           name: Set Ruby Version
           command: echo "ruby-2.4" > ~/.ruby-version
-      - run:
-          name: Update Homebrew
-          command: brew update
       - run:
           name: Install Swiftlint
-          command: brew install swiftlint
+          command: git clone git@github.com:realm/SwiftLint.git && cd SwiftLint && git submodule update --init --recursive && make install && cd ../
```

Comment:
Hey :)) Have you considered use [Mint](https://github.com/realm/SwiftLint#using-mint) here?

Reply:
That's a good point @LucianoPAlmeida. I just didn't want to add another dependency to CI in this PR. We may discuss potential updates to the CI script, as it needs refreshing anyways.

Reply:
Sure, that makes sense, I just raised the point because it appears to be a simpler script :) Also we could avoid having to clone all source and compile it, iff mint supports just download the binary(not sure if it does).